packages/flutter_tools/lib/src/project.dart

```diff
@@ -645,6 +645,11 @@ class AndroidProject {
   }
 
   AndroidEmbeddingVersion getEmbeddingVersion() {
+    if (isModule) {
+      // A module type's Android project is used in add-to-app scenarios and
+      // only supports the V2 embedding.
+      return AndroidEmbeddingVersion.v2;
```

Comment:
what about a project that was created with v1 and ran make-host-app-editable?

Reply:
Discussed on chat. If users generally consumed v1 in add-to-app, it would break in the next stable. Those plugins will need to be registered manually since our tools can't predict whether a v1 or v2 style consumption will be made in the host Android project. 

Document in https://github.com/flutter/flutter/issues/36491